lib/news/widgets/article_screen.dart

```diff
@@ -48,6 +49,17 @@ class ArticleScreen extends StatelessWidget {
       }),
     );
   }
+
+  AppBar _buildAppBar(BuildContext context, Article article) => AppBar(
+        elevation: 0,
+        backgroundColor: context.theme.scaffoldBackgroundColor,
+        iconTheme: IconThemeData(color: context.theme.contrastColor),
+        actions: <Widget>[
+          SizedBox(width: 8),
+          AccountButton(),
+          SizedBox(width: 8),
+        ],
+      );
```

Comment:
"People _writing_ code seem to love `=>`, but it’s very easy to abuse it and end up with code that’s hard to _read_. If your declaration is more than a couple of lines or contains deeply nested expressions—cascades and conditional operators are common offenders — do yourself and everyone who has to read your code a favor and use a block body and some statements." ~ [Effective Dart guidelines](https://dart.dev/guides/language/effective-dart/usage#consider-using--for-simple-members)